src/plugins/home/kibana.jsonc

```diff
@@ -7,7 +7,13 @@
     "server": true,
     "browser": true,
     "requiredPlugins": ["dataViews", "share", "urlForwarding"],
-    "optionalPlugins": ["usageCollection", "customIntegrations", "cloud", "guidedOnboarding"],
-    "requiredBundles": ["kibanaReact"]
+    "requiredBundles": ["kibanaReact"],
+    "optionalPlugins": [
+      "usageCollection",
+      "customIntegrations",
+      "cloud",
+      "guidedOnboarding",
+      "cloudExperiments"
```

Comment:
Is this being used? Or is it just a placeholder for now when A/B testing is implemented?

Reply:
Yes I believe that cloudExperiments will be necessary for launch darkly and I optimistically added it to the kibana.jsonc incase time permitted for me to figure it out before FF.